test/unittest/blas3/blas3_gemm_batched_test.cpp

```diff
@@ -44,7 +44,7 @@ GENERATE_GEMM_TEST(BatchGemm, BetaNonZeroLDMatch);
 
 const auto BetaNonZeroLDMultiplied =
     ::testing::Combine(::testing::Values(0),         // offset
-                       ::testing::Values(5),         // batch
+                       ::testing::Values(8),         // batch
```

Comment:
Does it not work with `5` anymore? Is there a restriction on being a multiple of 4?

Reply:
This is put to 8 to test the MiniBatchSize parameter. The kernel with MiniBatchSize=4 is used only if batch_size is divisible by 4. It still works with 5, there just wasn't any reason to keep both 5 and 8 I think.

 I'm still not sure whether we should add a specific test for MiniBatchSize, otherwise I can just add a comment explaining why we need a parameter divisible by 4.